rfc6962-bis.xml

```diff
@@ -1295,6 +1295,12 @@ but it is expected there will be a variety.
 	  <t hangText="Maximum Merge Delay">
 	    The MMD the log has committed to.
 	  </t>
+	  <t hangText="STH Frequency Period">
+	    The time, in seconds, over which <spanx style="verb">STH Frequency Count</spanx> is measured (see <xref target="STH"/>).
+	  </t>
+	  <t hangText="STH Frequency Count">
+	    The maximum number of STHs the log may produce in any <spanx style="verb">STH Frequency Period</spanx> (see <xref target="STH"/>).
+	  </t>
```

Comment:
OK.

Does the "Misbehaving Logs" section (currently 8.4) need to deal with the scenario of a log issuing more than "STH Frequency Count" STHs in any given "STH Frequency Period" ?